tools/noopfactory/src/test/resources/gen/NoOpGenericInterface.kt

```diff
@@ -1,29 +1,35 @@
 package com.example
 
 import kotlin.CharSequence
+import kotlin.Suppress
+import kotlin.Unit
 import kotlin.collections.List
 import kotlin.collections.Map
 import kotlin.collections.Set
 import kotlin.collections.emptyList
 import kotlin.collections.emptyMap
 import kotlin.collections.emptySet
 
+@Suppress(
+    "RedundantVisibilityModifier",
+    "RedundantUnitReturnType"
+)
 internal class NoOpGenericInterface<T : CharSequence> : GenericInterface<T> {
 
-    override fun doSomething() {
+    public override fun doSomething(): Unit {
```

Comment:
new version of `kotlin-poet` automatically adds `Unit` as well?

Reply:
yup